Team8923_2020/src/main/java/org/firstinspires/ftc/team8923_2020/CompetitionTeleOp.java

```diff
@@ -0,0 +1,28 @@
+package org.firstinspires.ftc.team8923_2020;
+
+import com.qualcomm.robotcore.eventloop.opmode.TeleOp;
+
+@TeleOp (name = "CompetitionTeleOp")
+public class CompetitionTeleOp extends MasterTeleOp{
+
+    @Override
+    public void runOpMode(){
+
+        initHardware();
+        waitForStart();
+
+        while(opModeIsActive()){
+
+            driveMecanum();
+
+            //runIntake();
+
+            //runLauncher();
+
+            //runWobbleGrabber();
+
+            idle();
+        }
+
+    }
+}
```

Comment:
Why did you comment out code in competition teleop? I thought you were just adding the autonomous classes